### ISSUE TITLE
Issues building Erlang

### ISSUE THREAD

**Issue by tswicegood** [conda/conda-build#149]
Seeing this when I try to build Erlang 17.1:

```
$ conda -V && conda build -V && conda build .
conda 3.5.5-99-gf8a075f
conda-build 1.5.0-41-g3d9418c
BUILD START: erlang-17.1-0
Fetching package metadata: .........
Solving package specifications: .
The following NEW packages will be INSTALLED:

    openssl:  1.0.1h-0
    readline: 6.2-2

Linking packages ...
[      COMPLETE      ] |################################################################################################################################| 100%
Source cache directory is: /Users/travis/anaconda/conda-bld/src_cache
Found source in cache: otp_src_17.1.tar.gz
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/Users/travis/anaconda/bin/conda-build", line 7, in <module>
    execfile(__file__)
  File "/Users/travis/Dropbox/work/continuum/conda-build/bin/conda-build", line 5, in <module>
    sys.exit(main())
  File "/Users/travis/Dropbox/work/continuum/conda-build/conda_build/main_build.py", line 109, in main
    args_func(args, p)
  File "/Users/travis/Dropbox/work/continuum/conda/conda/cli/main.py", line 182, in args_func
    args.func(args, p)
  File "/Users/travis/Dropbox/work/continuum/conda-build/conda_build/main_build.py", line 268, in execute
    build.build(m, verbose=not args.quiet, post=post)
  File "/Users/travis/Dropbox/work/continuum/conda-build/conda_build/build.py", line 274, in build
    source.provide(m.path, m.get_section('source'))
  File "/Users/travis/Dropbox/work/continuum/conda-build/conda_build/source.py", line 219, in provide
    unpack(meta)
  File "/Users/travis/Dropbox/work/continuum/conda-build/conda_build/source.py", line 68, in unpack
    tar_xf(src_path, WORK_DIR)
  File "/Users/travis/Dropbox/work/continuum/conda-build/conda_build/utils.py", line 45, in tar_xf
    t.extractall(path=dir_path)
  File "/Users/travis/anaconda/lib/python2.7/tarfile.py", line 2051, in extractall
    self.extract(tarinfo, path)
  File "/Users/travis/anaconda/lib/python2.7/tarfile.py", line 2088, in extract
    self._extract_member(tarinfo, os.path.join(path, tarinfo.name))
  File "/Users/travis/anaconda/lib/python2.7/posixpath.py", line 80, in join
    path += '/' + b
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe7 in position 54: ordinal not in range(128)
```

**Comment by stuertz** [comment]
I had the same Issue today, while building a recipe for pyslet and would like to suggest this fix:

```
323c323,326
<     t.extractall(path=dir_path)
---
>     if not PY3 and on_win:
>         t.extractall(path=dir_path.encode(codec))
>     else:
>         t.extractall(path=dir_path)
```

The problem may occur only on windows and only with py2. And it helped me :-)

**Comment by msarahan** [comment]
@stuertz thanks - that looks reasonable.  Please put up a PR if you don't mind.

**Comment by msarahan** [comment]
fixed by #1779.  Closing.
